### PR TITLE
Initial calendar work

### PR DESCRIPTION
**Pull request by djycheng**
calendar shows events and can add events, displaying events has problems

### CODE REVIEW

File: client/lib/routes.js [hunk 1]
```diff
@@ -34,7 +34,10 @@ Router.route('/admin', function() {
   this.render('admin');
 });
 
+Router.route('/calendar', function() {
+	this.render('calendar');
+});
+
 Router.configure({
-	layoutTemplate: 'ApplicationLayout',
-	notFoundTemplate: '404'
```
Comment (ajaska) [inline client/lib/routes.js:83]: wait let's keep the 404 template lol

File: client/views/test.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<head>
```
Comment (ajaska) [inline client/views/test.html]: Let's move css changes to the master css files

File: client/views/test.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/*
+Need:
+	How to log in - add isAdmin later
+	Modale/stylish popup for adding event
+	Drop down/window for displaying event info
+*/
+if (Meteor.isClient) {
```
Comment (ajaska) [inline client/views/test.js]: We should be the client -- probably remove this check for consistency across js view files :)

**Comment by ajaska** [comment]
Looks pretty good! Just some minor adjustments please :+1:

**Comment by djycheng** [comment]
Calendar template is now on home page, need admin privileges to add, edit, delete events, UI for event windows was fixed after updating my repository.  Please look over and add so we can show the premed people tomorrow.

**Comment by ajaska** [comment]
I'll merge this for now, but:
1. There's some weird things with variable scoping in `client/views/calendar.js`. Reminder that the javascript in any file in `client/views/` is included on ALL pages, so defining a bunch of global variables in there (i.e., at the root level or without `var`) is a little questionable.
2. Also why is it downgrading the package versions and adding a whole bunch of mixins?
3. I'm going to adjust the way the calendar is rendered in the template, so that it is rendered from the `landing` template (`landingPage.html`) instead of the top-level template (`ApplicationLayout.html`)

**Pull request merged**